http/headers/content-security-policy.json

```diff
@@ -588,7 +588,8 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": "33"
+                "version_added": "33",
+                "notes": "Before Firefox 58, this would be ignored in <code>Content-Security-Policy-Report-Only</code>"
               },
               "firefox_android": {
                 "version_added": "33"
```

Comment:
Was this bug only in Firefox for Desktop or is this also affecting Firefox on Android?

Reply:
I just tested on Android, and it's present there too. The change is in the core Gecko engine, which I assume is the same code for Android as it is for the desktop. I'll copy the note to apply to Firefox for Android as well.